Makefile

```diff
@@ -66,13 +66,13 @@ $(coreLib): buildFlags
 $(renderLib): buildFlags
 	"$(MAKE)" -C $(engineDir)/render CXXFLAGS="$(CXXFLAGS)"
 
-$(testApp): buildFlags $(utilsLib) $(coreLib)
+$(testApp): buildFlags $(utilsLib) $(coreLib) $(renderLib)
 	"$(MAKE)" -C $(testsDir) CXXFLAGS="$(CXXFLAGS)"
 
 $(exampleGameApp): buildFlags $(utilsLib) $(coreLib)
 	"$(MAKE)" -C $(examplesDir)/game CXXFLAGS="$(CXXFLAGS)"
 
-$(exampleRenderApp): buildFlags $(renderLib)
+$(exampleRenderApp): buildFlags $(utilsLib) $(renderLib)
```

Comment:
Again, unless you're testing `renderlib`, don't add it as a dip of `testApp`. Did you ever figure out why `utilslib` is needed here and if we can remove it?

Reply:
Yes, this is again an artifact from some previous coding. I'll get rid of this dependency.

Reply:
Fixed.